src/toolchain/web-guide.md

```diff
@@ -0,0 +1,135 @@
+<!--
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+
+# Intro to Web Builds
+
+Web builds are a fair bit more to get started with compared to native builds.
+This will be a complete guide on how to get things compiled.
```

Comment:
Just to be a bit more descriptive:

```suggestion
This will be a complete guide on how to properly compile your game using `gdext` to WebAssembly, which will allow you to use `gdext` in the web.
```

Reply:
I agree with the extension, but I think "properly" is not needed 😉

Reply:
I agree! Feel free to remove "properly"